README.md

```diff
@@ -799,6 +800,23 @@ $ dy scan
 
 # Misc
 
+## Development
+For development, we use `rustfmt` and `clippy` to maintain quality of source codes.
+Before starting, please ensure installation of both components;
```

Comment:
nit: grammar (it's a matter of taste though)
```suggestion
For development, we use `rustfmt` and `clippy` to maintain the quality of our source code.
Before starting, please ensure both of components are installed;
```

Reply:
Thank you. Your suggested change feels clear. But, I remove `of` after `both`.

Reply:
ah yes, that's what I meant. thanks!